packages/nextjs/utils/toast.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { CheckmarkIcon, ErrorIcon, toast as hotToast } from "react-hot-toast";
+import { XMarkIcon } from "@heroicons/react/20/solid";
+import { ExclamationTriangleIcon, InformationCircleIcon } from "@heroicons/react/24/solid";
+import Spinner from "../components/Spinner";
+
+type TPositions = "top-left" | "top-center" | "top-right" | "bottom-left" | "bottom-center" | "bottom-right";
+
+type TToastProps = {
+  message: React.ReactNode;
+  status: "success" | "info" | "loading" | "error" | "warning";
+  duration?: number;
+  icon?: string;
+  position?: TPositions;
+};
+
+type ToastOptions = {
+  duration?: number;
+  icon?: string;
+  position?: TPositions;
+};
+
+const hotToastIconStyles: React.CSSProperties = {
+  backgroundColor: "#7eaaef",
+  marginTop: "0.25rem",
+};
+
+const ENUM_STATUSES = {
+  success: <CheckmarkIcon style={hotToastIconStyles} />,
+  loading: <Spinner />,
+  error: <ErrorIcon style={hotToastIconStyles} />,
+  info: <InformationCircleIcon className="w-7  text-primary-focus" />,
+  warning: <ExclamationTriangleIcon className="w-7 text-primary-focus" />,
```

Comment:
Two things here:

1. Let's use hero icons (except for the spinner).
success =   CheckCircleIcon
error = ExclamationCircleIcon

2. Let's use different colors (rn all of them are blue) for each

We can use daisy colors (info / success / warning / errors)

![image](https://user-images.githubusercontent.com/2486142/208079579-1514ad1f-7d84-4ac2-ab38-51d31eda9e1d.png)